misc/extension_api_validation/4.1-stable.expected

```diff
@@ -293,3 +293,13 @@ GH-79527
 Validate extension JSON: Error: Field 'classes/ParticleProcessMaterial/properties/orbit_velocity_curve': type changed value in new API, from "CurveTexture" to "CurveTexture,CurveXYZTexture".
 
 Added accepted curve type from only CurveTexture to CurveXYZTexture.
+
+
+GH-83219
+--------
+Validate extension JSON: Error: Field 'classes/Node/methods/find_child/arguments': size changed value in new API, from 3 to 4.
+Validate extension JSON: Error: Field 'classes/Node/methods/find_parent/arguments': size changed value in new API, from 1 to 2.
+Validate extension JSON: Error: Hash changed for 'classes/Node/methods/find_parent', from 43F4625F to 241D75A5. This means that the function has changed and no compatibility function was provided.
+Validate extension JSON: Error: Hash changed for 'classes/Node/methods/find_parent', from 43F4625F to 241D75A5. This means that the function has changed and no compatibility function was provided.
```

Comment:
I think these lines should be removed if a compatibility method was provided.

Reply:
Alright, I removed the lines